addons/mrp/views/mrp_unbuild_views.xml

```diff
@@ -43,7 +43,7 @@
             </field>
         </record>
 
-       <record model="ir.actions.act_window" id="action_mrp_unbuild_moves">
+       <record model="ir.actions.act_window" id="action_mrp_unbuild_move_line">
```

Comment:
It's duplicated from https://github.com/odoo/odoo/blob/1d34e269dac2e0cafd85ad91a6240569b5b7282c/addons/mrp/views/mrp_unbuild_views.xml#L5-L8